src/render/index.html.eco

```diff
@@ -13,65 +13,109 @@ layout: default
       <div class="row">
-        <div class="col-sm-6">
-          <div class="knotx-movie">
-            <iframe width="560" height="315" src="https://www.youtube.com/embed/dBhgsmC78Yw?rel=0" frameborder="0" allowfullscreen></iframe>
-          </div>
+        <div class="col-sm-1">
+          <h2>System Admin</h2>
+          <p>
+            manages the platform
+          </p>
         </div>
-        <div class="col-sm-6">
-          <blockquote>
-            <p>Let’s imagine an online banking website containing different features like a chat box, exchange rates, stock data and user profile information. The site has a high performance characteristic, despite its complexity and target publishing channels.
-              All those features come from different providers/vendors having their own teams working in various modes, technologies and release cycles.
-              Knot.x connects all of them in a controlled and isolated way, preventing any undesired interferences. It combines asynchronous programming principles and message-driven architecture providing a scalable platform for modern sites.
-              Knot.x connects all of the above to create a unified customer experience.
-            </p>
-          </blockquote>
+        <div class="col-sm-2">
+          <h2>Experience Manageable Integrations + Continuous Replatforming</h2>
+          <p>
+            combines various systems in a controlled and isolated way, preventing any
+            undesired interferences; gradually embed new systems, CMSes on your website
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Resilience</h2>
+          <p>
+            keeps your platform always available. Even if a module breaks, it can redeploy
+            themselves into a new location/server
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Microservices 2.0</h2>
+          <p>
+            fine-grained modular integration logic (modules with precise responsibility) with
+            manageable contracts, allowing them to be scaled separately without affecting the
+            codebase
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Scaling</h2>
+          <p>
+            integration logic can be precisely scaled vertically and horizontally, the platform
+            provides load balancing for free (the platform implements reactive principles)
+          </p>
+        </div>
+        <div class="col-sm-1">
+          <h2>Extendable & Polyglot</h2>
+          <p>
+            extension points supporting multiple programming languages
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Channels / Users</h2>
+          <p>
+            the IoT is coming, modern platforms have to support both different channels like tablets,
+            mobiles, computers and other applications. With Knot.x you can address the need of
+            any channel, SPA, etc.
```

Comment:
>          the IoT is coming, modern platforms have to support both different channels like tablets,
>          mobiles, computers and other applications. With Knot.x you can address the need of
>            any channel, SPA, etc.

how about:

_the Internet of Things is coming. Modern platforms have to be made available to user via a variety of channels like tabletes, mobiles, computers, as well as connected devices and other applications. Knot.x gives you the tools to reach your customers regardless of the approach._

Also, what's an SPA? Avoid using abbreviations unless they've been preiously introduced.

Reply:
fixed